BuildResidentialHPXML/measure.rb

```diff
@@ -1411,6 +1411,61 @@ def arguments(model)
     arg.setDefaultValue(30)
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('mech_vent_is_shared_system', true)
+    arg.setDisplayName('Mechanical Ventilation: Is Shared System')
+    arg.setDescription('Whether the mechanical ventilation is a shared system. If true, assumed to serve all the units in the building.')
+    arg.setDefaultValue(false)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('shared_mech_vent_flow_rate', false)
```

Comment:
I suggest calling this `shared_mech_vent_in_unit_flow_rate` instead, so it's not confused with the existing `mech_vent_flow_rate` variable.